README.md

```diff
@@ -84,7 +91,7 @@ Other components (Prometheus, Kafka, Grafana) can be omitted from the installati
 ```
 
 
-#### Bring Your Own ElasticSearch
+### Install with your own ElasticSearch
```

Comment:
Nice. Ya, BYOX is fun for labeling parties - but, I think this is better for our situation 👍